contrib/bootstrap.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+wget https://raw.githubusercontent.com/saltstack/salt-bootstrap/6080a18e6c7c2d49335978fa69fa63645b45bc2a/bootstrap-salt.sh -O /tmp/bootstrap-salt.sh
+chmod 700 /tmp/bootstrap-salt.sh
+apt-get update && apt-get -y install python-pip git python-dev
+pip install boto
+pip install gnupg
+cd /tmp
+git clone https://github.com/ministryofjustice/bootstrap-salt.git
```

Comment:
`git clone --depth 1 --branch $TAG https://github.com/ministryofjustice/bootstrap-salt.git` will make it clone just the one revision it needs rather than all history ever